File: vetor/exercicio3.py

numeros = [0] * 100

for i in range(100):
    numeros[i] = int(input(f"Digite o número {i+1}: "))

x = int(input("Digite o valor de x: "))

existe = False
for i in range(100):
    if numeros[i] == x:
        existe = True
        break
1
if existe:
    print(f"O valor {x} existe no conjunto.")
else:
    print(f"O valor {x} não foi encontrado no conjunto.")

File: vetor/exercicio3.c
#include <stdio.h>

int main() {
    int vetor[100];
    int x;
    int encontrado = 0;

    printf("Digite 100 números inteiros:\n");
    for (int i = 0; i < 100; i++) {
        printf("Número %d: ", i + 1);
        scanf("%d", &vetor[i]);
    }

    printf("Digite o valor a ser buscado (x): ");
    scanf("%d", &x);

    for (int i = 0; i < 100; i++) {
        if (vetor[i] == x) {
            encontrado = 1;
            break;
        }
    }

    if (encontrado) {
        printf("O valor %d existe no conjunto.\n", x);
    } else {
        printf("O valor %d não foi encontrado no conjunto.\n", x);
    }

    return 0;
}
